dev-requirements.txt

```diff
@@ -0,0 +1 @@
+twine==4.0.2
```

Comment:
I would just add it to `requirements.txt`, since I think today that file is already a mix of dev packages and runtime packages.